### PR TITLE
Reduced parameter count for macro M0_BTREE_OP_SYNC_WITH_RC.

### PR DESCRIPTION
**Pull request by shashank-parulekar**
Signed-off-by: Shashank Parulekar <shashank.parulekar@seagate.com>

### CODE REVIEW

**Review by NikhilBirgade — APPROVED** [review]

File: btree/btree.c [hunk 1]
```diff
@@ -5152,24 +5152,19 @@ static void ut_basic_tree_oper(void)
 	/** Create temp node space*/
 	temp_node = m0_alloc_aligned((1024 + sizeof(struct nd)), 10);
 	btree = m0_alloc(sizeof *btree);
-	rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op.bo_op,
-				      m0_btree_create(temp_node, 1024,
-						      &btree_type, nt, &b_op, tx
-						      ), &b_op.bo_sm_group,
-				      &b_op.bo_op_exec);
+	rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op, m0_btree_create(temp_node, 1024,
+						      &btree_type, nt, &b_op,
```
Comment (RadhaGulhane13) [inline btree/btree.c:5156]: alignment.

File: btree/btree.c [hunk 2]
```diff
@@ -5199,17 +5193,14 @@ static void ut_basic_tree_oper(void)
...
 
 	/** Create a new btree */
 	temp_node = m0_alloc_aligned((1024 + sizeof(struct nd)), 10);
-	rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op.bo_op,
-				      m0_btree_create(temp_node, 1024,
-						      &btree_type, nt, &b_op, tx
-						      ), &b_op.bo_sm_group,
-				      &b_op.bo_op_exec);
+	rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op, m0_btree_create(temp_node, 1024,
+						      &btree_type, nt, &b_op,
```
Comment (RadhaGulhane13) [inline btree/btree.c:5202]: alignment

File: btree/btree.c [hunk 3]
```diff
@@ -5240,17 +5230,15 @@ static void ut_basic_tree_oper(void)
 	 * ToDo: This condition needs to be uncommented once the check for
 	 * node_is_valid is properly implemented in btree_open_tick.
 	 *
-	 * rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op.bo_op,
+	 * rc = M0_BTREE_OP_SYNC_WITH_RC(&b_op,
 	 *                             m0_btree_open(invalid_addr, 1024, &btree,
```
Comment (RadhaGulhane13) [inline btree/btree.c:5234]: alignment

File: btree/btree.c [hunk 4]
```diff
@@ -5259,10 +5247,9 @@ static void ut_basic_tree_oper(void)
...
```
Comment (RadhaGulhane13) [inline btree/btree.c:5251]: alignment

**Review by ydb242 — APPROVED** [review]

**Review by RadhaGulhane13 — APPROVED** [review]

**Pull request merged**